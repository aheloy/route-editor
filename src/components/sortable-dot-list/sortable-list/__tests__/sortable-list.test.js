import React from 'react';

import SortableList from '../sortable-list';


const items = [
  {
    text: '1'
  },
  {
    text: '2'
  },
  {
    text: '3'
  }
];
const onSort = () => 0;
const onDelete = () => 0;


test('render a SortableList', () => {
  const wrapper = shallow(
    <SortableList
      items={items}
      onSort={onSort}
      onDelete={onDelete}
    />
  );
  expect(wrapper).toMatchSnapshot();
});
