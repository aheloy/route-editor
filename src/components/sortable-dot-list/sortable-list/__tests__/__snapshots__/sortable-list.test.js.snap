// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a SortableList 1`] = `
<styled.ul>
  <Sortable
    items={
      Array [
        Object {
          "text": "1",
        },
        Object {
          "text": "2",
        },
        Object {
          "text": "3",
        },
      ]
    }
    key="0"
    onDelete={[Function]}
    onSortItems={[Function]}
    sortId={0}
  >
    1
  </Sortable>
  <Sortable
    items={
      Array [
        Object {
          "text": "1",
        },
        Object {
          "text": "2",
        },
        Object {
          "text": "3",
        },
      ]
    }
    key="1"
    onDelete={[Function]}
    onSortItems={[Function]}
    sortId={1}
  >
    2
  </Sortable>
  <Sortable
    items={
      Array [
        Object {
          "text": "1",
        },
        Object {
          "text": "2",
        },
        Object {
          "text": "3",
        },
      ]
    }
    key="2"
    onDelete={[Function]}
    onSortItems={[Function]}
    sortId={2}
  >
    3
  </Sortable>
</styled.ul>
`;
